generative_ai/inference/example_syntax.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import vertexai
+
+from vertexai.language_models import TextEmbedding
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+vertexai.init(project=PROJECT_ID, location="us-central1")
+
+
+def generate_response() -> TextEmbedding:
```

Comment:
Combines the samples for streaming and non-streaming [responses](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference#syntax). Adds imports and fixes the syntax, so the sample can be copied and ran, but leaves off all details to keep the sample very concise.